enterprise/ha/src/main/java/org/neo4j/kernel/ha/SlaveTransactionCommitProcess.java

```diff
@@ -54,10 +58,21 @@ public long commit( TransactionRepresentation representation, LockGroup locks, C
         try
         {
             RequestContext context = requestContextFactory.newRequestContext( representation.getLockSessionId() );
+            logger.debug( "Committing on master with " + context + " and " + representation );
```

Comment:
This will flood the messages.log with log statements and change the whole timing of commits and whatnot. It should not serialize them per se behind the file lock since we already guard with the logical log lock, but it will affect the way races manifest. Plus, it will create really big logs and make execution slower.